hw2/task01/task01.py

```diff
@@ -9,9 +9,14 @@
     5) Find most common non ascii char for document
 """
 
-import string
+import sys
+from typing import Generator
 from unicodedata import category
 
+codepoints = range(sys.maxunicode + 1)
+punctuation = {c for i in codepoints if category(c := chr(i)).startswith("P")}
```

Comment:
If you have a way to check if a single char is punctuation, why do you need a set for it?